### PR TITLE
Create createCustomLobby.py

### PR DESCRIPTION
**Pull request by XHXIAIEIN**

### CODE REVIEW

File: examples/createCustomLobby.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+#---------------------------------------------
+
+connector = Connector()
+
+@connector.ready
+async def connect(connection): 
+  print(connection.address)
+  await get_lockfile(connection)
+  await get_summonerdata(connection)
+  await creat_lobby(connection)
```
Comment (sousa-andre) [inline examples/createCustomLobby.py:93]: The coroutine does not exist. Typo.

**Review by sousa-andre — CHANGES_REQUESTED** [review]
Can you always use single quotes instead of double quotes for strings?

File: examples/createCustomLobby.py (hunk 1 above)
Comment (XHXIAIEIN) [reply examples/createCustomLobby.py:93]: Sorry, I'll fix it

Comment (XHXIAIEIN) [reply examples/createCustomLobby.py]: > The coroutine does not exist. Typo.

I modified it, please check again

**Pull request merged**